### ISSUE TITLE
Question: Is the project abandoned, and do you need help?

### ISSUE THREAD

**Issue by dlebech** [Adori/fastapi-cloud-tasks#10]
Hi @ofpiyush, are you still maintaining this library?

The library is very useful, and I'm not actually missing features, but it *is* a little difficult to use the library with the latest developments in Pydantic and FastAPI, and it would be nice to either get an official patch with https://github.com/Adori/fastapi-cloud-tasks/pull/8 

Thank you for this library, and if you need any help with maintenance, let me know.

**Comment by ofpiyush** [comment]
I don't have an active project using it and I don't work at the company anymore.

Would really appreciate help in maintaining and feedback from active users on PRs. 🙏 

@niyeradori can someone at Adori pick it?

**Comment by dlebech** [comment]
Thanks for you quick reply @ofpiyush :pray: 

In terms of feedback, that's probably better for another thread, but I can tell you briefly that I'm using [a fork](https://github.com/svaletech/fastapi-cloud-tasks/) with the only change being #8 cherry-picked, and that makes the library usable with the latest updates to fastapi/pydantic.

**Comment by ofpiyush** [comment]
Yea we should get the changes merged in.

cc: @niyeradori

**Comment by yogeshdecodes** [comment]
Hi @dlebech and @ofpiyush 
Are the proposed changes in [#8](https://github.com/Adori/fastapi-cloud-tasks/pull/8) backward compatible?
Additionally, do we need to upgrade the required FastAPI version to 0.100.0 to accommodate the Pydantic v2 changes proposed in #8?

**Comment by dlebech** [comment]
@yogeshdecodes No, I don't think that #8 is backwards compatible and would be a breaking change for this library and should bump the major version... And yes, fastapi >=0.100.0 would then be required and should probably be specified in setup.py file (which it isn't right now).

But we're a bit off topic now :slightly_smiling_face: My question was more about whether anyone is going to take this package forward, or whether I should continue to use my own fork...

**Comment by yogeshdecodes** [comment]
@dlebech I wanted to reassure you that the project is not abandoned. In fact, we're using this library in many projects internally at Adori, which speaks to its value and utility within our workflows.

I'm relatively new to managing this project, but I'll be taking on the role of supporting its maintenance going forward. While our current priorities mean that we might not be focusing on introducing new features, we are committed to fixing bugs and ensuring the library remains compatible with the latest versions.

We genuinely appreciate your offer to help with the library's maintenance and any feedback you might have.

**Comment by dlebech** [comment]
@yogeshdecodes that's good to hear! :tada:
